### ISSUE TITLE
Inconsistent parameter naming Macro __DEFINE_HAS_LVAL_MMBR_FUNC_ 1

### ISSUE THREAD

**Issue by AnantaYudica** [AnantaYudica/basic#9]
```
#include "Macro/HasLValueMemberFunction.h"

#define __DEFINE_HAS_MMBR_FUNC_TMPL_PARAM_LIST_TPL1(\
    __T_NAME__,\
    __T_RET_NAME__,\
    __T_ARGS_NAME__)\
typename __T_NAME__, typename T1, typename __T_RET_NAME__,\
    typename... __T_ARGS_NAME__

__DEFINE_HAS_LVAL_MMBR_FUNC_(HasLValMmbrFunc2, _HasLValMmbrFunc2, T, Tr, 
    Targs, TPL1, T_NAME_,,,,, PROTO_FUNC_T_NAME_,, PROTO_FUNC_T_RET_NAME_,, Foo1);
```
parameter macro `__DEFINE_HAS_LVAL_MMBR_FUNC_ ` using name `__TMPL_PARAM_LIST_POSTFIX__`
but call back implementation using name `__DEFINE_HAS_MMBR_FUNC_TMPL_PARAM_LIST_`

Similar with #4

**Comment by AnantaYudica** [comment]
Fixed in v0.1-beta-2

**Comment by AnantaYudica** [comment]
Not a bug, Macro `__DEFINE_HAS_LVAL_MMBR_FUNC_` will be deleted in v0.1-beta-2.
